### PR TITLE
Resolved bug in Edge: 

### PR DESCRIPTION
**Pull request by Palakagrawal28**
1)Removed underlines from contacts in footer
2)Also corrected the hover effect on links in footer

### CODE REVIEW

File: Footer.html [hunk 1]
```diff
@@ -31,17 +31,17 @@
...
                         <div class="footer-contacts">
                             <i class="footer-icon-color fa fa-phone primary-color" aria-hidden="true"></i>
-                            <p class=" link-footer-services">571-364-7032</p>
+                            <a class=" link-footer-services">571-364-7032</a>
                         </div>
 
                         <div class="footer-contacts">
                             <i class="footer-icon-color fa fa-print primary-color " aria-hidden="true"></i>
-                            <p class="link-footer-services">703-650-9593</p>
+                            <a class="link-footer-services">703-650-9593</a>
```
Comment (amitjain24524) [inline Footer.html]: Use Phone Number as Call Link ex. `<a href="tel:703-650-9593">703-650-9593</a>`

File: Footer.html [hunk 2]
```diff
@@ -31,17 +31,17 @@
...
                         </div>
 
                         <div class="footer-contacts">
                             <i class="footer-icon-color fa fa-mobile primary-color " aria-hidden="true"></i>
-                            <p class="link-footer-services">703-338-4567</p>
+                            <a class="link-footer-services">703-338-4567</a>
```
Comment (amitjain24524) [inline Footer.html]: same

File: Footer.html (hunk 1 above)
Comment (Palakagrawal28) [reply Footer.html]: done

File: Footer.html (hunk 2 above)
Comment (Palakagrawal28) [reply Footer.html]: done

**Pull request merged**